docs/project/list-of-diagnostics.md

```diff
@@ -90,6 +90,9 @@ The PR that reveals the implementation of the `<IncludeInternalObsoleteAttribute
 |  __`SYSLIB0033`__ | Rfc2898DeriveBytes.CryptDeriveKey is obsolete and is not supported. Use PasswordDeriveBytes.CryptDeriveKey instead. |
 |  __`SYSLIB0034`__ | CmsSigner(CspParameters) is obsolete and is not supported. Use an alternative constructor instead. |
 |  __`SYSLIB0035`__ | ComputeCounterSignature without specifying a CmsSigner is obsolete and is not supported. Use the overload that accepts a CmsSigner. |
+|  __`SYSLIB0036`__ | AssemblyName.HashAlgorithm has been deprecated and is not supported. |
```

Comment:
All 3 should use the same ID: https://github.com/dotnet/runtime/issues/59061#issuecomment-924248304

Reply:
Does not see any other obsoletion with the same id, wondering if the messages should be merged into one or it can have several rows with the same id CC @jeffhandley